code/modules/spacepods/spacepod.dm

```diff
@@ -73,6 +73,8 @@
 	pr_int_temp_processor = new /datum/global_iterator/pod_preserve_temp(list(src))
 	pr_give_air = new /datum/global_iterator/pod_tank_give_air(list(src))
 	equipment_system = new(src)
+	equipment_system.installed_modules = list()
```

Comment:
The issue that was occurring in game (which can be checked currently in the server) is that for whatever reason, it isn't treating it as a list. I have the line you pointed out in my current build, and until I defined it in the main code, it wasn't working as a list, but rather seemed to be just a singular reference.